test/specs/calculator.add.multiply.spec.js

```diff
@@ -0,0 +1,158 @@
+const { expect } = require('chai');
+const { Calculator } = require('../../app/calculator');
+
+// eslint-disable-next-line max-len
+describe('It should test Calculator class method \'add\' positive scenarios', () => {
+  let calculator;
+  beforeEach(() => {
+    calculator = new Calculator();
+  });
+  afterEach(() => {
+    calculator = null;
+  });
+  it('It should test the calculation of the natural positive numbers', () => {
+    expect(calculator.add(865, 45, 239)).to.be.equal(1149);
+  });
+  it('It should test the calculation with the negative number', () => {
+    expect(calculator.add(-356, 125)).to.be.equal(-231);
+  });
+  it('It should test the calculation with whole numbers', () => {
+    expect(calculator.add(548, 0)).to.be.equal(548);
+  });
+  it('It should test the calculation with fractions', () => {
+    expect(calculator.add(3 / 8, 4 / 8)).to.be.equal(7 / 8);
+  });
+  it('It should test the calculation with the rational numbers', () => {
+    // eslint-disable-next-line max-len
+    expect(calculator.add(-245.969, 138.354)).to.be.equal(-107.61499999999998);
+  });
+});
+
+// eslint-disable-next-line max-len
+describe('It should test Calculator class method \'add\' negative scenarios', () => {
+  let calculator;
+  beforeEach(() => {
+    calculator = new Calculator();
+  });
+  afterEach(() => {
+    calculator = null;
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' is null an error is thrown', () => {
+    expect(() => calculator.add(null, 5, 6)()).to.throw('Unable to calculate, [null] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'increment\' is not defined(null) an error is thrown', () => {
+    expect(() => calculator.add(68, null)()).to.throw('Unable to calculate, [null] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' and \'increment\' are not defined(null) an error is thrown', () => {
+    expect(() => calculator.add(null, null)()).to.throw('Unable to calculate, [null] is not a number');
```

Comment:
It's a bad practice to test a scenario with 2 invalid arguments. Please correct.

Reply:
Why? Due to the fact that our methods can receive any amount of any values I don't think it's an invalid check.

Reply:
In this case, can not be sure of the exact reason for your test being failed.